adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -896,7 +896,7 @@ def _wait_for_msg(self, timeout=0.1):
 
         # Block while we parse the rest of the response
         self._sock.settimeout(timeout)
-        if res in [None, b""]:
+        if res in [None, b"", bytearray(b'\x00')]:
```

Comment:
Does it work if this is just `b"\x00"`? Not sure you need to create a bytearray.

Reply:
I've just tried:

    if res in [None, b"", b"\x00"]:

And it appears to work as expected. Decent matrix framerates, no `MemoryError`'s etc.